README.md

```diff
@@ -70,3 +103,20 @@ Just change the `ports` configuration.
 cd frontend
 npm start
 ```
+
+### Note about backend-frontend proxy
+
+Take note that even though the frontend React server is available via http://localhost:3000, you should be entering the React app via http://localhost:4000 instead.
+
+This is because the backend (:4000) proxies to the frontend server (:3000) and is required for Facebook login to work.
+
+Specifically, the request to GET '/oauth/login' does not work when using React server (:3000)
+
+## Development using Twilio Voice
+
+Twilio requires a public URL to confirm calls via webhook. The webhook URL can be set up with https://ngrok.com/ for now.
+
+In Twilio's console, update the REQUEST URL with your ngrok `https` url.
+`Voice / TwiML / TwiML Apps / Dev backend > REQUEST URL`
+
+Note that this method only works for one developer at a time.
```

Comment:
We can add a new TwiML app for each dev, then we won't mess with each other's development :)